chef.py

```diff
@@ -0,0 +1,156 @@
+"""
+PRATHAM Open School is organized as follow:
+- There is top level set of topics (e.g. Mathematics, English, Science, ...)
+- Each topic has subtopics (e.g. Geometry, Algebra, ...)
+- Each subtopic has lessons (e.g. Triangle, Circle, Polygons, ...)
+- Finally, each subtopic has contents like videos, pdfs and html5 files.
+
+TODO: hn/Fun and hn/Story do not follow this structure. For now, they
+are not being scraped.
+"""
+
+from bs4 import BeautifulSoup
+import re
+import requests
+import urllib
+
+from le_utils.constants import licenses
+from ricecooker.classes.files import VideoFile, HTMLZipFile, WebVideoFile, ThumbnailFile
+from ricecooker.classes.nodes import ChannelNode, HTML5AppNode, TopicNode, VideoNode, DocumentNode, ExerciseNode
+from ricecooker.config import LOGGER
+from ricecooker.utils.caching import CacheForeverHeuristic, FileCache, CacheControlAdapter
+
+cache = FileCache('.webcache')
+basic_adapter = CacheControlAdapter(cache=cache)
+forever_adapter = CacheControlAdapter(heuristic=CacheForeverHeuristic(),
+                                      cache=cache)
+session = requests.Session()
+session.mount('http://', basic_adapter)
+session.mount('https://', basic_adapter)
+session.mount('http://www.prathamopenschool.org', forever_adapter)
+session.mount('https://www.prathamopenschool.org', forever_adapter)
+
+BASE_URL = 'http://www.prathamopenschool.org'
+# In debug mode, only one channel is downloaded.
+DEBUG_MODE = False
+
+
+def construct_channel(*args, **kwargs):
+    channel = ChannelNode(
+        title='Pratham Open School',
+        source_domain=BASE_URL,
+        source_id='pratham-open-school',
```

Comment:
The `source_id` needs to be different for each language (since each will have its own channel, and this is used to calculate the channel ID). Perhaps `'pratham-open-school-{}'.format(kwargs['language'])`